common/src/main/java/com/wynntils/functions/BombFunctions.java

```diff
@@ -108,15 +108,15 @@ public Float invalidValue() {
         }
     }
 
-    public static class BombRemainingTimeFunction extends BombFunctionBase<Long> {
+    public static class BombEndTimeFunction extends BombFunctionBase<Time> {
         @Override
-        public Long processInfo(BombInfo info) {
-            return info.getRemainingLong();
+        public Time processInfo(BombInfo info) {
+            return Time.of(info.endTime());
         }
 
         @Override
-        public Long invalidValue() {
-            return -1L;
+        public Time invalidValue() {
+            return Time.of(-1L);
```

Comment:
There is a `Time.NONE`, I think it is better to use it. (In fact, as the comment says, using -1 to represent an invalid time is not really ideal. If we use Time.NONE everywhere instead of Time.of(-1), this is easy to change if we decide we need to in the future.)